staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_discovery_controller.go

```diff
@@ -112,7 +122,8 @@ func (c *DiscoveryController) sync(version schema.GroupVersion) error {
 					Version:      v.Name,
 				})
 			}
-			if v.Name == version.Version {
+
+			if v.Name == gvt.version {
```

Comment:
version vs Version? Is there any reason why it was Capitalized V?

Reply:
The upper-case Version is from the type schema.GroupVersion, which is a type used in a lot of locations. So its members are public, in upper cases.

gvt is of type GroupVersionTenant, which is a type I define to use in this controller only. Therefore I made the members private, in lower cases.